src/zpotrf_wrapper.c

```diff
@@ -104,6 +105,33 @@ static void destroy_workspace(void *_ws, void *_n)
     free(ws);
     (void)_n;
 }
+
+/* Find all devices */
+static void parsec_find_nb_devices(int **dev_index, int *nb) {
```

Comment:
parsec_context_query is insufficient because we also need to convert cuda indices to parsec indices, tracking issue 
https://github.com/ICLDisco/parsec/issues/660

For the sake of not halting progress lets proceed with this pr then.